tests/test_packageAlerts.py

```diff
@@ -569,7 +614,12 @@ def testRun_without_produce(self, mock_server_check):
                 pixelPoint,
                 geom.Extent2I(self.cutoutSize, self.cutoutSize),
                 cutout.getPhotoCalib(),
-                1234)
+                1234,
+                cutoutType='science')
```

Comment:
Why isn't the cutout type "difference" here? You're comparing to the stand-in for the difference image, right?

Reply:
Will swap to difference. I wasn't thinking about that since the result is the same whether or not it is science or difference, but for consistency I will change it to difference.